subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/cntr.c

```diff
@@ -13,7 +13,7 @@
 #define LOG_MODULE_NAME bt_ctlr_cntr
 #include "common/log.h"
 #include "hal/debug.h"
-#include "nrf_rtc.h"
+#include <nrfx/hal/nrf_rtc.h>
```

Comment:
Why not hal/nrf_rtc.h here as well?

Reply:
To avoid confusion with the `hal/` directory [in the bluetooth controller](https://github.com/zephyrproject-rtos/zephyr/tree/master/subsys/bluetooth/controller/hal). The inclusion right above (`#include "hal/debug.h"`) is done from this other HAL.